src/main/java/com/minecolonies/coremod/util/ColonyUtils.java

```diff
@@ -59,11 +59,11 @@ public static boolean hasNewSubscribers(@NotNull final Set<EntityPlayerMP> oldSu
      * @return a tuple with the required corners.
      */
     public static Tuple<Tuple<Integer, Integer>, Tuple<Integer, Integer>> calculateCorners(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "isMirrored" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)